### DISCUSSION TITLE
Style a component on global level

### DISCUSSION THREAD

**Discussion by sqveeze** [mantinedev/mantine#813] [Q&A] [answered]
Hello,

I would like to ask how to style for example an input on global level, so like every input will use a style that set border radius to 0. And if i want, then i can override in component level by sx prop. SX prop is cool, but i don't want to write borderRadius: 0 for every input in sx.

**Comment by rtivital** [comment, marked as answer]
You can add context styles to any component use `styles` prop on MantineProvider – https://mantine.dev/theming/mantine-provider/#styles-on-mantineprovider

**Reply by sqveeze** [reply]
> You can add context styles to any component use `styles` prop on MantineProvider – https://mantine.dev/theming/mantine-provider/#styles-on-mantineprovider

Thank you very much! I was not looking at that page. Instead i was trying to find the solution in the extend theme section: https://mantine.dev/theming/extend-theme

There should be a mention at least to the page you linked 👍 

Thank you again! Thank you for your work with Mantine
